org-docs/CHARTER.md

```diff
@@ -46,19 +46,24 @@ In addition to the method set out in section 2.2, the membership of a Steering C
 
 **4.4 Effect of Removal**. Upon the removal of a non-Sponsored Steering, that position on the Steering Committee is permanently removed. When a Sponsored Steering Committee member is removed for any reason, their position on the Steering Committee becomes vacant and may be replaced concurrently with removal or separately at a later time by the Sponsoring entity by written notice to the Steering Committee. A Sponsored seat may be permanently abandoned and be removed by written notice from the Sponsoring entity to the Steering Committee.
 
-## 5. Trademarks
+## 5. Maintainers
```

Comment:
I thought we were planning on just adding this to the readme, to prevent needing to ratify a full charter amendment.